docs/source/install.rst

```diff
@@ -44,26 +44,27 @@ binary dependencies from ``apt`` by running:
 
 .. code-block:: bash
 
-   $ sudo apt-get install libatlas-base-dev python-dev gfortran pkg-config libfreetype6-dev
+   $ sudo apt-get install libatlas-base-dev python-dev gfortran pkg-config libfreetype6-dev hdf5
 
 On recent `RHEL-derived`_ derived Linux distributions (e.g. Fedora), the
 following should be sufficient to acquire the necessary additional
 dependencies:
 
 .. code-block:: bash
 
-   $ sudo dnf install atlas-devel gcc-c++ gcc-gfortran libgfortran python-devel redhat-rpm-config
+   $ sudo dnf install atlas-devel gcc-c++ gcc-gfortran libgfortran python-devel redhat-rpm-config hdf5
 
 On `Arch Linux`_, you can acquire the additional dependencies via ``pacman``:
 
 .. code-block:: bash
 
-   $ pacman -S lapack gcc gcc-fortran pkg-config
+   $ pacman -S lapack gcc gcc-fortran pkg-config hdf5
 
 There are also AUR packages available for installing `ta-lib
 <https://aur.archlinux.org/packages/ta-lib/>`_, an optional Zipline dependency.
 Python 2 is also installable via:
 
+
```

Comment:
Would you update the macOS instructions to include brew installing `hdf5` too?

Reply:
Sure!